websites/T/Tanks.gg/presence.ts

```diff
@@ -0,0 +1,117 @@
+const presence = new Presence({
+		clientId: "1003195428642238464",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/jjvJi16.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href } = window.location;
+
+	if (pathname === "/") presenceData.details = "Browsing the home page";
+	else if (document.location.pathname.startsWith("/tank/")) {
+		presenceData.details = "Viewing a tank";
+		presenceData.state = `${
+			document.querySelector(".header-tank > h1").childNodes[0].textContent
+		}`;
+		presenceData.buttons = [
+			{
+				label: "View Tank",
+				url: href,
+			},
+		];
+	} else if (document.location.pathname.startsWith("/list/stats")) {
+		presenceData.details = "Browsing tanks by stats";
+		presenceData.state = `Sorting by: ${
+			document.querySelector(
+				"#content > div > div.container-fluid.main-container > div > div.filter > div.dropdown.stat > button"
+			).textContent
+		}`;
+		presenceData.buttons = [
+			{
+				label: "View Page",
+				url: href,
+			},
+		];
+	} else if (document.location.pathname.startsWith("/techtree/")) {
+		presenceData.details = "Viewing the tech tree";
+		presenceData.state = `${
+			document.querySelectorAll(".current")[0].innerHTML
+		}`;
```

Comment:
If you're getting the first one from a querySelectorAll then use a normal querySelector cuz that always gets the first one
```suggestion
		presenceData.state = `${
			document.querySelector(".current").innerHTML
		}`;
```